src/main/java/marquez/core/exceptions/NamespaceException.java

```diff
@@ -0,0 +1,3 @@
+package marquez.core.exceptions;
+
+public class NamespaceException extends Exception {}
```

Comment:
This Exception isn't used - should it be in the review?

Reply:
I think this was introduced in one of your commits? 

```
$ find . -name NamespaceException.java | xargs git log
commit c91a6fcb5502804e5c84a954e16badced5131ad2
Author: ashulmanWeWork <41963174+ashulmanWeWork@users.noreply.github.com>
Date:   Wed Oct 31 15:23:26 2018 -0700

    Add basic namespace resources and data object (#116)
```

Reply:
I must have gotten picked up in the rebase

Reply:
ah, this test is no longer valid, which is probably why I disabled. will mark ignored -- feel free to re-enable if you use these tests for the endpoints